src/utils.rs

```diff
@@ -193,26 +170,6 @@ mod tests {
         });
     }
 
-    #[test]
-    fn handle_none_argument() {
-        let path_opt: Option<PathBuf> = None;
-        let path = path_opt.unwrap_or_else(default_wallet_path);
-        validate_wallet_path(&path).unwrap();
-        let default_path = default_wallet_path();
-        assert_eq!(path, default_path)
-    }
-
-    #[test]
-    fn handle_relative_path_argument() {
```

Comment:
Does the CLI still support relative paths?

Reply:
Ahh good catch! I'll re-add this. I think I removed this thinking it was already tested elsewhere, but it doesn't look like it is.